Scripts/Mac/BuildGVFSForMac.sh

```diff
@@ -59,9 +59,12 @@ xcodebuild -configuration $CONFIGURATION -workspace $NATIVEDIR/GVFS.Native.Mac.x
 USERNOTIFICATIONDIR=$VFS_SRCDIR/GVFS/GVFS.Notifications/VFSForGit.Mac
 USERNOTIFICATIONPROJECT="$USERNOTIFICATIONDIR/VFSForGit.xcodeproj"
 USERNOTIFICATIONSCHEME="VFS For Git"
-updateAppVersionCmd="(cd \"$USERNOTIFICATIONDIR\" && /usr/bin/xcrun agvtool new-marketing-version \"$VERSION\")"
-echo $updateAppVersionCmd
-eval $updateAppVersionCmd
+# don't update version number for developer builds
+if [[ $VERSION != 0* ]]; then
```

Comment:
Our releases to mac early adopters are numbered 0.6.xxxxx.y. Can we make this test whether a version was explicitly provided to the script instead?

Reply:
Updated to only run version updater tool when an explicit version is provided.